src/Components/Header.js

```diff
@@ -0,0 +1,22 @@
+import { Component } from 'react';
+import Navbar from 'react-bootstrap/Navbar'
+class Header extends Component {
+
+    render() {
```

Comment:
Hey there!

Some feedback here:

Indent the class a bit so it isn't right upon the imports 👍  The Header component should have an <h1> inside (see assignment instructions).